src/test/java/com/iexec/sms/admin/AdminServiceTests.java

```diff
@@ -179,21 +178,46 @@ void withSQLException() {
     @Test
     void shouldDeleteBackup() throws IOException {
         final String backupFileName = "backup.sql";
-        final Path tmpFile = Files.createFile(tempStorageLocation.toPath().resolve(backupFileName));
+        final File backupDatabaseFile = new File(tempStorageLocation.getPath() + "/" + backupFileName + AdminService.AES_KEY_FILENAME_EXTENSION);
+        final File backupAesKeyFile = new File(tempStorageLocation.getPath() + "/" + backupFileName + AdminService.AES_KEY_FILENAME_EXTENSION);
+
         assertAll(
+                () -> assertThat(adminService.createBackupFile(tempStorageLocation.getPath(), "backup.sql")).isTrue(),
+                () -> assertThat(backupDatabaseFile).exists(),
+                () -> assertThat(backupAesKeyFile).exists(),
                 () -> assertThat(adminService.deleteBackupFileFromStorage(tempStorageLocation.getPath(), backupFileName)).isTrue(),
-                () -> assertThat(tmpFile.toFile()).doesNotExist(),
-                () -> assertThat(memoryLogAppender.contains("Successfully deleted backup")).isTrue()
+                () -> assertThat(backupDatabaseFile).doesNotExist(),
+                () -> assertThat(backupAesKeyFile).doesNotExist(),
+                () -> assertThat(memoryLogAppender.contains("Database delete process done")).isTrue(),
+                () -> assertThat(memoryLogAppender.contains("AES Key delete process done")).isTrue()
         );
     }
 
     @Test
-    void shouldFailToDeleteWhenBackupFileMissing() throws IOException {
-        final String backupStorageLocation = tempStorageLocation.getCanonicalPath();
-        assertThatExceptionOfType(FileSystemNotFoundException.class)
-                .isThrownBy(
-                        () -> adminService.deleteBackupFileFromStorage(backupStorageLocation, "backup.sql")
-                ).withMessageContaining(AdminOperationError.DATABASE_BACKUP_FILE_NOT_EXIST.toString());
+    void shouldFailToDeleteWhenOneBackupFilesMissing() {
+        final String backupFileName = "backup.sql";
+        final File backupDatabaseFile = new File(tempStorageLocation.getPath() + "/" + backupFileName + AdminService.AES_KEY_FILENAME_EXTENSION);
```

Comment:
Error on the backup filename:
```suggestion
        final File backupDatabaseFile = new File(tempStorageLocation.getPath() + "/" + backupFileName);
```